### PR TITLE
Lc723

### PR DESCRIPTION
**Pull request by Houjue92**
solution for lc723

### CODE REVIEW

File: week2/lc723_crushCandy.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+                board[k][i] = 0;
+            } else {
+                board[k][i] = temp.pop();
+            }    
+        }
+    }    
+    return board;
+}
+
+const mark = (board, found) => {
```
Comment (gnotnay) [inline week2/lc723_crushCandy.js:45]: What's the time complexity of this mark function? O(n * m * m)?

File: week2/lc723_crushCandy.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
+/**
+ * @param {number[][]} board
+ * @return {number[][]}
+ */
+var candyCrush = function(board) {
+    // let m = board.length;
```
Comment (gnotnay) [inline week2/lc723_crushCandy.js:6]: nit: delete

File: week2/lc723_crushCandy.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
+/**
+ * @param {number[][]} board
+ * @return {number[][]}
+ */
+var candyCrush = function(board) {
+    // let m = board.length;
+    // let n = board[0].length;
```
Comment (gnotnay) [inline week2/lc723_crushCandy.js:7]: nit: delete

File: week2/lc723_crushCandy.js [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+        let cnt = 0;
+        let temp = []
+        for(let j = 0; j < board.length; j++) {
+            if(board[j][i] <= 0) {
+                cnt++;  
+            } else {
+                temp.push(board[j][i]);
+            }    
+        }
+        temp.reverse();
```
Comment (gnotnay) [inline week2/lc723_crushCandy.js:33]: why reverse?

**Review by jasonchu237 — APPROVED** [review]
Good job, thanks.

**Review by lyzlalala — APPROVED** [review]

**Review by gnotnay — APPROVED** [review]

**Pull request merged**